### PR TITLE
bug fix for issues/4

### PR DESCRIPTION
**Pull request by edithau**
bug fix for https://github.com/opentracing/opentracing-ruby/issues/4

@itsderek23 could you please review?  I edited `.gitignore` for RubyMine.   Hope that's ok.

### CODE REVIEW

**Comment by itsderek23** [comment]
Thx @edithau! I'll check this out this week.

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 *.gem
 *.rbc
+.idea/
```
Comment (mwear) [inline .gitignore]: You could achieve the same thing by adding this to `.git/info/exclude` in your local checkout.

File: lib/opentracing/span.rb [hunk 2]
```diff
@@ -44,7 +44,7 @@ def get_baggage_item(key)
     # @param event [String] event name for the log
```
Comment (mwear) [inline lib/opentracing/span.rb]: The `event` parameter has been removed, but the documentation remains.

File: .gitignore (hunk 1 above)
Comment (edithau) [reply .gitignore]: 👍

File: lib/opentracing/span.rb (hunk 2 above)
Comment (edithau) [reply lib/opentracing/span.rb]: good catch!  Thanks!

**Comment by edithau** [comment]
@itsderek23 @mwear I'm going to close this PR and replace it with a PR for https://github.com/opentracing/opentracing-ruby/issues/22.  If u prefer to keep this one, please let me know!

**Comment by edithau** [comment]
replaced by #23

**Pull request closed without merging**